src/model_signing/hashing/file.py

```diff
@@ -192,11 +193,24 @@ def digest_name(self) -> str:
 
     @override
     def compute(self) -> hashing.Digest:
-        # https://github.com/python/typeshed/issues/2166
-        # pytype: disable=wrong-arg-types
-        digest = hashlib.file_digest(self._fd, self._algorithm)
-        # pytype: enable=wrong-arg-types
-        return hashing.Digest(self.digest_name, digest.digest())
+        if sys.version_info >= (3, 11):
+            # https://github.com/python/typeshed/issues/2166
+            # pytype: disable=wrong-arg-types
+            digest = hashlib.file_digest(self._fd, self._algorithm)
+            # pytype: enable=wrong-arg-types
+            return hashing.Digest(self.digest_name, digest.digest())
+        else:
+            # Polyfill for Python 3.10
```

Comment:
TIL about `polyfill`. Just another term for backport?

Optional: The Go dev in me would say drop the `else` to reduce indentation one block, but dont know if this applies in Python so free to ignore.

Reply:
It's from Javascript, it's for anything that covers a gap in some implementation (like a browser missing some feature). In our case it's the same as backport.

Good idea on dropping the `else`. I shortened the link too, so now we don't have a line that is too long.